graphReconstruct/graphReconstruct.go

```diff
@@ -2,230 +2,95 @@ package graphReconstruct
+		column.AlignNodes[root.Name] = append(column.AlignNodes[root.Name], newNode)
+		id += 1
 	}
-	return true
+	return id
 }
 
-//set up Stored list for each node in the tree with probability of each base
-func SetState(node *expandedTree.ETree, position int) {
-	if node.Left != nil && node.Right != nil {
-		SetState(node.Left, position)
-		SetState(node.Right, position)
-		if allZero(node.Left.Stored) && allZero(node.Right.Stored) {
-			log.Fatal("no Stored values passed to internal node")
-		}
-
-		for i := 0; i < 4; i++ {
-			sum := 0.0
-			for j := 0; j < 4; j++ {
-				for k := 0; k < 4; k++ {
-					sum = sum + Prob(i, j, node.Left.BranchLength)*node.Left.Stored[j]*Prob(i, k, node.Right.BranchLength)*node.Right.Stored[k]
-				}
-			} //branch length (transition probability) times the probability of a base appearing given the context of the lower tree nodes
-			node.Stored[i] = sum
-		}
-	} else if node.Left != nil {
-		SetState(node.Left, position)
-		if node.Left.Stored == nil {
-			log.Fatal("no Stored values passed to internal node, left branch")
-		}
-		for i := 0; i < 4; i++ {
-			sum := 0.0
-			for j := 0; j < 4; j++ {
-				sum = sum + Prob(i, j, node.Left.BranchLength)*node.Left.Stored[j]
-			} //branch length (transition probability) times the probability of a base appearing given the context of the lower tree nodes
-			node.Stored[i] = sum
-		}
-	} else if node.Right != nil {
-		SetState(node.Right, position)
-		if node.Right.Stored == nil {
-			log.Fatal("no Stored values passed to internal node, right branch")
-		}
-		for i := 0; i < 4; i++ {
-			sum := 0.0
-			for k := 0; k < 4; k++ {
-				sum = sum + Prob(i, k, node.Right.BranchLength)*node.Right.Stored[k]
+//BuildEdges connects the nodes of a species' graph that are stored in GraphColumns
+//func BuildEdges
+//start without prob
+//loop through species in column, go through all nodes
+//check if that node's seq matches the seq of the ancestor node without an edge
+//make the next of that node the same as this node's next
+//FindAncSeq creates a graph from the node records stored in GraphColumns and then calls PathFinder and seqOfPath to determine the most likley seq of the ancestor before assigning that
+//seq to the Fasta field of the ancestors tree node
+//func FindAncSeq will loop through aligncolumns and build a single graph of all of the nodes that belong to the ancestor species after edges are created
+//run PathFinder on the graph for the anc, run seqOfPath, then turn that to a fasta for that node of the tree
+
+//seqOfPath takes in a graph and a path specified by the Node IDs and returns the seq of the path through the graph
+func seqOfPath(g *simpleGraph.SimpleGraph, path []uint32) []dna.Base {
+	var seq []dna.Base
+	var foundInGraph = false
+	for n := 0; n < len(g.Nodes); n++ {
+		for p := 0; p < len(path); p++ {
```

Comment:
Right inside this loop over 'p', I think we need to set FoundInGraph to false each time so that foundInGraph is by default false for each element of path until that element is found